update_hof.py

```diff
@@ -75,8 +75,18 @@
 """.replace("\n", "")
 
 credit_entries = {
+    "3ce577ba2fed1b2decbeb3eb4e466f8d":"Alesandro Ortiz",
+    "0c239777752cc449e91df93984e4c0c8":"Irwan",
+    "4cc72613ad5eb1dc54b6acf0be217aed":"Mattias Jacobsson",
+    "f72d4bcd18f0526f8faa4f781513c91f":"Hafiizh",
```

Comment:
what is the significance of the IDs? do they act like merge keys? how are they generated?

Reply:
They are mapping of the reporter's email address to their name/twitter/url.  They ensure that we can add those credit details to the first bug bounty attachment (or manually in this script if it's a HOF only person) and all future instances of the user will have their credit properly applied.

When the script is processing bugs and sees a new user with information in the bug bounty attachment that is not present in the script, the script actually edits itself to add the entries.

The actual value is an HMAC(secret, email) where secret is stored on a private bug.  This ensures that if someone wants to report bugs anonymously, we aren't SHA-ing their real email in a public place where it could be brute-forced.

Reply:
do you have instructions on how to run the script? I also noticed that the list of web products does not include `Websites` which is used to classify some web bounty reports, would that matter?

Reply:
`./update_hof.py -a API_KEY [-c/-w] -y 2021 -q 4 -f bug-bounty-hof/[client.yml/web.yml]`

-c and client.yml for client; -w and web.yml for web.

As for web products - no.  We define what is a client bug more carefully and anything hat isn't a client bug is a web bug, so it catches everything else.  Also, when the script is run it generates a bug list one should paste into the browser and double check things like products/components.